actual-budget/docker-compose.yml

```diff
@@ -0,0 +1,15 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: actual-budget-web-1
+      APP_PORT: 5006
+      PROXY_AUTH_ADD: "false"
+  
+  web:
+    image: actualbudget/actual-server:latest@sha256:19ef6ce46b2fe9ce61178648e9e13831b29040fd6a4c80130fb3f7c2bd9dfc5a
```

Comment:
Hey @arguser , Please use the actual `version` you're trying to use rather than the `latest` tag.